### DISCUSSION TITLE
Getting Error 2103 while installing otel agent on windows server.

### DISCUSSION THREAD

**Discussion by RPooja08** [signalfx/splunk-otel-collector#1935] [General]
Getting Error 2103 while installing otel agent on windows server.
![image](https://user-images.githubusercontent.com/68017992/187860533-fbb91db9-9fca-495b-96fd-283c630c30c2.png)

**Comment by atoulme** [comment]
This seems to indicate that you attempted to install the collector using a per-user installation from the system account. Only machine installations are permitted from the system account. Read more [here](https://www.itninja.com/question/help-with-msi-2103).
